tests/components/UserActions/ClickedDocumentList.spec.ts

```diff
@@ -9,6 +9,7 @@ import { UserActionType } from '../../../src/rest/UserProfilingEndpoint';
 describe('ClickedDocumentList', () => {
     const TEST_CLICKS = generate(20, i => {
         const document = Fake.createFakeResult();
+        document.uri = `document${Math.random()}`;
```

Comment:
We talked about it in a former review with @erocheleau: what's the point of Random in tests. That's, imho, undesirable and useless complexity.

Reply:
The random part prevent depending on a specific string value.

Reply:
Personally I would rather have documentTestUri that a random, that's some over-engineering imho.
What do you think @tedre191, @erocheleau ?